src/widgets/User/UserHeader/UserInfo.tsx

```
@@ -0,0 +1,62 @@
+  const { id } = router.query
+  const userId = id ? Number(id) : NaN
+  const { data, loading } = useGetUserInfoQuery({
+    variables: {
+      id: +userId,
+    },
+  })
+  const userInfo = data ? data.getUser : ({} as User)
+  const usersAvatar =
+    (userInfo &&
+      userInfo.profile &&
+      userInfo.profile.avatars &&
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)